cli/internal/auth/apikeys/apikeys.go

```diff
@@ -135,7 +157,7 @@ func list(ctx context.Context, orgTitle, projectTitle string) error {
 		return err
 	}
 
-	tbl := table.New("Name", "Owner", "Created At", "Secret")
+	tbl := table.New("Name", "Owner", "A", "B", "Created At", "Secret")
```

Comment:
what rows or fields do "A" and "B" indicate here?

Reply:
removing. not sure, but probably some leftover from github copilot recommendation.